util/codec/codec.go

```diff
@@ -989,7 +989,7 @@ func (decoder *Decoder) DecodeOne(b []byte, colIdx int, ft *types.FieldType) (re
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		if ft.Decimal != types.UnspecifiedLength && frac > ft.Decimal {
+		if ft.Decimal != types.UnspecifiedLength && frac != ft.Decimal {
```

Comment:
Why change this code? If we don't change it, we also can pass all the tests as well.

Reply:
No, now, you will fail in the test `TestModifyColumnWithDecimal`

Reply:
It's my mistake. Ignore it.